availability.php

```diff
@@ -76,6 +76,6 @@
     <form action="availability.php" method="post">
       ' . daily_matrix ( $date, $users ) . '
     </form>
-    ' . print_trailer ( false, true, true );
+    ' . print_trailer ( 0, 1, 1 );
```

Comment:
Hey, Bruce.  I think we're on the same page here mostly.  I'm hesitant about changes to large numbers of files because it's time-consuming to test and validate (especially without any unit tests or CI in place).